openslides_backend/models/assert_migration_index.py

```diff
@@ -0,0 +1,37 @@
+from datastore.shared.di import injector
```

Comment:
I don't think this belongs in the `models` folder, but rather in the `migrations` folder

Reply:
ok, I'll move it.